esterni/piano-di-qualifica/components/introduzione.tex

```diff
@@ -24,7 +24,7 @@ \subsection{Scopo del prodotto}%
 % sub:scopo_del_prodotto (end)
 \subsection{Glossario}%
 \label{sub:glossario}
-Al fine di rendere il documento più chiaro possibile, i termini che possono assumere un significato ambiguo sono evidenziati (i.e., \glossario{client}) e riportati in \textit{Glossario1.0.0.pdf} accompagnati da una definizione.
+Al fine di rendere il documento più chiaro possibile, i termini che possono assumere un significato ambiguo sono evidenziati (i.e., \glossario{client}) e riportati in \textit{Glossario (versione \versione) } accompagnati da una definizione.
```

Comment:
_\textit{Glossario (versione \versione) }_
La versione attuale del glossario è 0.1.3, quindi sarà **\textit{Glossario0.1.3}** (con o senza .pdf)

Reply:
con questo comando la versione dei documenti citati rimane sempre coerente con la versione globale del progetto

Reply:
si questa era una cosa che avevo scritto in https://github.com/GruppOne/project-docs/issues/117#issuecomment-593116278

Facendo come dici tu, ci toccherebbe aggiornare manualmente tutti i riferimenti ogni volta che cambiamo qualcosa, invece così anche se non è una soluzione perfetta almeno basta farlo una volta e dimenticarsene

Reply:
Buono a sapersi